### DISCUSSION TITLE
Keine IP von DHCP

### DISCUSSION THREAD

**Discussion by brekemeier** [tbnobody/OpenDTU#2076] [General]
OpenDTU neustes Version: DHCP setzt keine IP

**Comment by brekemeier** [comment]
OpenDTU setzt keine DHCP-IP: System/Netzwerk: 0.0.0.0

**Reply by ghost** [reply]
Bei mir schon.
(Kleine Korrektur, die openDTU setzt keine IP Adresse, sie erhält die von deinem Router, wenn der per DHCP-Adressen vergibt und noch freie hat, sowie neue Geräte im Netzwerk zulässt.)
Ich brauchte mal schnell etwas um mein WLAN zu testen, es ist in openDTU ja so etwas wie eine Empfangsanzeige drin. Es gibt bestimmt effektiveres, mir reicht es auf die Schnelle.

Gerade eben einen nagelneuen esp32 mit der opendtu-generic_esp32.factory.bin geflasht.
Kurz mit dem AP verbunden meine SSID und PW eingetragen (beziehen per DHCP belassen).
Innerhalb von 1-2 Sekunden im WLAN erreichbar. (Kanal 1 am als LAN-Brücke laufenden 1750E Mesh-Repeater an einer FB7590) Bekommen hat die DTU eine x.x.x.5.

**Comment by brekemeier** [comment]
Informatiker, ist schon klar

Am 18. Juni 2024 21:09:28 MESZ schrieb "J. G." ***@***.***>:

**Comment by mroenne2022** [comment]
Wie schon richtig gesagt wurde **erhält** die DTU eine IP per DHCP ( setzen/verteilen tut sie keine), dazu muss der entsprechende Punkt in den Einstellungen natürlich eingeschaltet sein.
![Unbenannt](https://github.com/tbnobody/OpenDTU/assets/114778920/5382d18f-2c66-4217-815d-70c0469a17f4)

**Comment by brekemeier** [comment]
OK, danke, morgen früh 
Gruss Bernhard 

Am 18. Juni 2024 21:27:53 MESZ schrieb mroenne2022 ***@***.***>:

**Reply by mroenne2022** [reply]
Müsste eigentlich klappen, hab mehrere DTUn schon gebaut, noch nie gab es ein Problem mit IP per DHCP. 
Ansonsten ist tatsächlich im Router das Problem zu suchen, z.B. wie schon gesagt wurde das der Router überhaupt neue
Geräte zulässt.

**Comment by brekemeier** [comment]
Hat er

Am 18. Juni 2024 21:27:53 MESZ schrieb mroenne2022 ***@***.***>:

**Comment by brekemeier** [comment]
OK, ich mach ein Reset,morgen früh. Er vergibt keine IP, ansonsten sehe ich mir das Release an. Hab ja alles installiert 

Am 18. Juni 2024 21:40:06 MESZ schrieb mroenne2022 ***@***.***>:
